modules/news/news.service.js

```diff
@@ -14,9 +14,6 @@ class NewsService {
   }
 
   async updateNews(id, news) {
-    if (await this.checkNewsExist(news)) {
```

Comment:
Why have you removed this method call? It is not related to the your merge request, please, revert it

Reply:
Changed!